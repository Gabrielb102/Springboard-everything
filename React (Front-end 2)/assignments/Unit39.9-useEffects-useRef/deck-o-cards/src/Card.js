import { useEffect } from "react";

const Card = ({imgSrc, keepDrawing, drawFunc}) => {

    return (
        <img src={imgSrc} />
    )
}

export default Card;