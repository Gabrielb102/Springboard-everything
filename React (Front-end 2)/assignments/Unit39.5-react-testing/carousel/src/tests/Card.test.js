import Card from "../Card";
import { render, fireEvent } from "@testing-library/react";

describe("Tests for the Card component", () => {
    it("renders", () => {
        render(<Card />);
    });

    it("consistently renders the right HTML", () => {
        const { asFragment } = render(<Card />);
        expect(asFragment).toMatchSnapshot();
    });
})