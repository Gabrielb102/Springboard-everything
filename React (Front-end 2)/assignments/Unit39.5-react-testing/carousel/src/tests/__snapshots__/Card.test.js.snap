// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the Card component consistently renders the right HTML 1`] = `[Function]`;
